Cargo.lock

```diff
@@ -13132,7 +13134,7 @@ version = "1.6.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "97fee6b57c6a41524a810daee9286c02d7752c4253064d0b05472833a438f675"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
Do you know what caused the downgrade of the version?

Reply:
Very likely due to the patching for `PureStake/frontier`. Should be reverted once I remove the patch.